### ISSUE TITLE
GANet2 and GANet7

### ISSUE THREAD

**Issue by xubin1994** [feihuzhang/GANet#20]
Hi,I want to verify the results in the text,but I want to know the reult of GANet2 or GANet7  was directly taking the corresponding output on GANet15?Or it needs to be trained separately with its own error?
Another thing I am more curious about is that GANet has very good generalization capabilities, what is the key to the perfect performance? SGA layer or LGA layer?

**Comment by feihuzhang** [comment]
1) Need to retrain these different models.
2) How do you measure the generalization ability? I didn’t test it and was not sure about  the generalization abilities. Did you test the model on other dataset?

**Comment by xubin1994** [comment]
> 1. Need to retrain these different models.
> 2. How do you measure the generalization ability? I didn’t test it and was not sure about  the generalization abilities. Did you test the model on other dataset?

Yes, I test the model on MiddleBurry dataset with the model sceneflow_epoch_10.pth.Moreover，I get some passive images pairs through Intel D435,and the GANet Deep produce nice disparity maps.

**Comment by feihuzhang** [comment]
> Yes, I test the model on MiddleBurry dataset with the model sceneflow_epoch_10.pth.Moreover，I get some passive images pairs through Intel D435,and the GANet Deep produce nice disparity maps.

I did some comparisons these days. Model trained on sceneflow dataset also generalize better on KITTI compared with others. I found it's mainly related with the SGA layer. SGA is a differentiable SGM and seems to be able to improve the model's generalization ability. I will add some analysis and update the arxiv paper. Thank you so much for help pointing out such an advantage!!!
